services/static.js

```diff
@@ -258,6 +287,24 @@ Static.getStaticImage = function(config) {
     );
   }
 
+  if (
+    config.padding !== undefined &&
+    (config.position !== 'auto' && config.position.bbox === undefined)
+  ) {
+    throw new Error(
+      'Padding can only be used with auto or with a bounding box as the position.'
+    );
+  }
+  if (
+    config.position.bbox !== undefined &&
+    (config.position.pitch !== undefined ||
+      config.position.bearing !== undefined ||
+      config.position.zoom !== undefined)
+  ) {
+    throw new Error(
+      'The bounding box parameter cannot be used with additional location parameters, bearing, or pitch'
+    );
+  }
```

Comment:
[nit] If we properly assert on the shape of the `position` argument above, then this check should be mostly unnecessary.